---
title: Benchmarking single-cell RNA-sequencing protocols for cell atlas projects
author: Zijian
date: '2020-04-10'
slug: benchmarking-single-cell-rna-sequencing-protocols-for-cell-atlas-projects
categories:
  - cancer
tags:
  - Single cell, RNA seq
---



<div id="title" class="section level1">
<h1>Title</h1>
<blockquote>
<p>Benchmarking single-cell RNA-sequencing protocols for cell atlas projects</p>
</blockquote>
</div>
<div id="citation" class="section level1">
<h1>Citation</h1>
<blockquote>
<p>Mereu, E., Lafzi, A., Moutinho, C. et al.Â Benchmarking single-cell RNA-sequencing protocols for cell atlas projects. Nat Biotechnol (2020). <a href="https://doi.org/10.1038/s41587-020-0469-4" class="uri">https://doi.org/10.1038/s41587-020-0469-4</a></p>
</blockquote>
</div>
<div id="abtract" class="section level1">
<h1>Abtract</h1>
<blockquote>
<p>Single-cell RNA sequencing (scRNA-seq) is the leading technique for characterizing the transcriptomes of individual cells in a sample. The latest protocols are scalable to thousands of cells and are being used to compile cell atlases of tissues, organs and organisms. However, the protocols differ substantially with respect to their RNA capture efficiency, bias, scale and costs, and their relative advantages for different applications are unclear. In the present study, we generated benchmark datasets to systematically evaluate protocols in terms of their power to comprehensively describe cell types and states. We performed a multicenter study comparing 13 commonly used scRNA-seq and single-nucleus RNA-seq protocols applied to a heterogeneous reference sample resource. Comparative analysis revealed marked differences in protocol performance. The protocols differed in library complexity and their ability to detect cell-type markers, impacting their predictive value and suitability for integration into reference cell atlases. These results provide guidance both for individual researchers and for consortium projects such as the Human Cell Atlas.</p>
</blockquote>
</div>
<div id="take-home-message" class="section level1">
<h1>Take home message</h1>
<blockquote>
<p>Single-cell genomics provides an unprecedented view of the cellular makeup of complex and dynamic systems</p>
</blockquote>
<blockquote>
<ul>
<li>the identification of new cell types</li>
<li>the fine-grained description of cell plasticity in dynamic systems,</li>
<li>improved the scale, accuracy and sensitivity of scRNA-seq methods</li>
</ul>
</blockquote>
<blockquote>
<p>The HCA project aims to advance our understanding of tissue function and to serve as a reference for defining variation in human health and disease.</p>
</blockquote>
<blockquote>
<p>A comprehensive comparison of available scRNA-seq protocols will benefit both large- and small-scale applications of scRNA-seq</p>
</blockquote>
<blockquote>
<p>cryopreservation is suitable for single-cell transcriptomic studies of these tissue type</p>
</blockquote>
</div>
