<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Transposons: The Jumping Genes | Just for your understanding</title>
    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    
  </head>

  <body>
    <nav>
    <ul class="menu">
      
      <li><a href="/">Home</a></li>
      
      <li><a href="/about/">About</a></li>
      
      <li><a href="/categories/">Categories</a></li>
      
      <li><a href="/tags/">Tags</a></li>
      
      <li><a href="/index.xml">Subscribe</a></li>
      
    </ul>
    <hr/>
    </nav>

<div class="article-meta">
<h1><span class="title">Transposons: The Jumping Genes</span></h1>
<h2 class="author">Leslie A. Pray</h2>
<h2 class="date">2020/04/17</h2>
</div>

<main>
<p><strong>Transposable elements, or &ldquo;jumping genes&rdquo;, were first identified by Barbara McClintock more than 50 years ago. Why are transposons so common in eukaryotes, and exactly what do they do?</strong></p>
<blockquote>
<p>Transposable elements (TEs)make up approximately 50% of the human genome and up to 90% of the maize genome</p>
</blockquote>
<blockquote>
<p>class 1 TEs (Retrotransposons) and class 2 TEs (DNA Transposons)</p>
</blockquote>
<h1 id="dna-transposons">DNA Transposons</h1>
<ol>
<li>All complete or &ldquo;autonomous&rdquo; class 2 TEs encode the protein transposase, which they require for insertion and excision</li>
<li>Class 2 TEs are characterized by the presence of terminal inverted repeats. One of the roles of terminal inverted repeats is to be recognized by transposase</li>
<li>In addition, all TEs in both class 1 and class 2 contain flanking direct repeats</li>
</ol>
<h1 id="retrotransposons">Retrotransposons</h1>
<ol>
<li>There are two major types of class 1 TEs: LTR retrotransposons, which are characterized by the presence of long terminal repeats (LTRs) on both ends; and non-LTR TEs, which lack the repeats.</li>
<li>Both the LINE1, or L1, and Alu genes represent families of non-LTR TEs. L1 elements average about 6 kilobases in length. In contrast, Alu elements average only a few hundred nucleotides, thus making them a short interspersed transposable element, or SINE</li>
<li>L1 makes up an estimated 15%–17% of the human genome</li>
</ol>
<h1 id="autonomous-and-nonautonomous-transposons">Autonomous and Nonautonomous Transposons</h1>
<p>Autonomous TEs can move on their own, while nonautonomous elements require the presence of other TEs in order to move</p>
<h1 id="what-jumping-genes-do-besides-jump">What Jumping Genes Do (Besides Jump)</h1>
<ul>
<li>Much of what a transposon does depends on where it lands. Landing inside a gene can result in a mutation, as was discovered when insertions of L1 into the factor VIII gene caused hemophilia.  L1 in the APC genes in colon cancer cells but not in the APC genes in healthy cells in the same individuals</li>
</ul>
<blockquote>
<h1 id="silencing-and-transposons">Silencing and Transposons</h1>
<ul>
<li>As opposed to L1, most TEs appear to be silent</li>
<li>Some silenced TEs are inactive because they have mutations that affect their ability to move</li>
<li><strong>others are perfectly intact and capable of moving but are kept inactive by epigenetic defense mechanisms such as DNA methylation, chromatin remodeling, and miRNAs</strong></li>
<li><strong>in recent years, however, researchers have begun to wonder whether certain TEs might themselves have a role in epigenetic silencing</strong></li>
</ul>
<h1 id="transposons-can-encode-sirnas-that-mediate-their-own-silencing">Transposons Can Encode siRNAs That Mediate Their Own Silencing</h1>
<ul>
<li>the 17% of the human genome that is encoded by L1-related sequences, only about 100 active L1 elements remain. Moreover, research suggests that even these few remaining active transposons are inhibited from jumping in a variety of ways that go beyond epigenetic silencing</li>
<li>siRNAs that interfere with L1 activity are derived from the 5′ untranslated region (5′ UTR) of L1 LTRs. Specifically, the 5′ UTR of the L1 promoter encodes a sense promoter that transcribes the L1 genes, as well as an antisense promoter that transcribes an antisense RNA. Yang and Kazazian (2006) demonstrated that this results in homologous sequences that can hybridize, thereby forming a double-stranded RNA molecule that can serve as a substrate for RNAi</li>
</ul>
<h1 id="transposons-are-not-always-destructive">Transposons Are Not Always Destructive</h1>
<ul>
<li>transposons can drive the evolution of genomes by facilitating the translocation of genomic sequences, the shuffling of exons, and the repair of double-stranded breaks</li>
<li>Insertions and transposition can also alter gene regulatory regions and phenotypes</li>
</ul>
</blockquote>
<h1 id="transposons-the-jumping-geneshttpswwwnaturecomscitabletopicpagetransposons-the-jumping-genes-518"><a href="https://www.nature.com/scitable/topicpage/transposons-the-jumping-genes-518/">Transposons: The Jumping Genes</a></h1>

</main>

  <footer>
  <script src="//yihui.name/js/math-code.js"></script>
<script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML"></script>

<script async src="//yihui.name/js/center-img.js"></script>

  
  <hr/>
  © <a href="https://zijzhang.netlify.com">Zijian Zhang</a> 2018 &ndash; 2019 | <a href="https://github.com/zijzhang">Github</a>
  
  </footer>
  </body>
</html>

