<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>The index to evaluate gene expression in RNA-seq | Just for your understanding</title>
    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <script data-ad-client="ca-pub-5931609864546180" async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
  </head>

  <body>
    <nav>
    <ul class="menu">
      
      <li><a href="/">Home</a></li>
      
      <li><a href="/about/">About</a></li>
      
      <li><a href="/categories/">Categories</a></li>
      
      <li><a href="/tags/">Tags</a></li>
      
      <li><a href="/index.xml">Subscribe</a></li>
      
    </ul>
    <hr/>
    </nav>

<div class="article-meta">
<h1><span class="title">The index to evaluate gene expression in RNA-seq</span></h1>
<h2 class="author">Zijian Zhang</h2>
<h2 class="date">2018/07/21</h2>
</div>

<main>
<h1 id="backgroud">Backgroud</h1>
<p>One commen quesion is wether one certain gene is expresed in one cell line or tissue or organ in human or mouse. for wet lab, qPCR or antibody-based assays can be performed with commercial antibodies. if no antibody is available, How should we do and this data is indispensable.</p>
<h1 id="rpkm-fkpm-and-tpm">RPKM, FKPM and TPM</h1>
<blockquote>
<p>RPKM (Reads Per Kilobase Million) or FPKM (Fragments Per Kilobase Million). However, TPM (Transcripts Per Kilobase Million) is now becoming quite popular.</p>
</blockquote>
<blockquote>
<p>RPKM:</p>
<ol>
<li>Count up the total reads in a sample and divide that number by 1,000,000 – this is our “per million” scaling factor.</li>
<li>Divide the read counts by the “per million” scaling factor. This normalizes for sequencing depth, giving you reads per million (RPM)</li>
<li>Divide the RPM values by the length of the gene, in kilobases. This gives you RPKM.</li>
</ol>
</blockquote>
<blockquote>
<p>FPKM is very similar to RPKM. RPKM was made for single-end RNA-seq, where every read corresponded to a single fragment that was sequenced. FPKM was made for paired-end RNA-seq. With paired-end RNA-seq, two reads can correspond to a single fragment, or, if one read in the pair did not map, one read can correspond to a single fragment. The only difference between RPKM and FPKM is that FPKM takes into account that two reads can map to one fragment</p>
</blockquote>
<blockquote>
<p>TPM is very similar to RPKM and FPKM. The only difference is the order of operations. Here’s how you calculate TPM:</p>
</blockquote>
<blockquote>
<ol>
<li>Divide the read counts by the length of each gene in kilobases. This gives you reads per kilobase (RPK).</li>
<li>Count up all the RPK values in a sample and divide this number by 1,000,000. This is your “per million” scaling factor.</li>
<li>Divide the RPK values by the “per million” scaling factor. This gives you TPM.</li>
</ol>
</blockquote>
<h1 id="z-score">Z-score</h1>
<blockquote>
<p>The z-score gives you the numer of standard-deviations that a value is away from the mean of all the values in the same group (= same gene).</p>
</blockquote>
<blockquote>
<p>A z-score of -3 for the gene X in sample A means that this value is 3 standard-deviations lower than the mean of the values for gene X in all the samples (A,B,C&hellip;).
When a gene is differentially expressen in two groups (like treated and control or disiesed and healthy), then the z-scores of samples will be (mostly) positive in one group and (mostly) negative in the other group.</p>
</blockquote>
<blockquote>
<p>The z-score is a signal-to-noise ratio. Large absolute z-scores (=large positive or negative values) is therefore not a direct estimate of the effect (i.e. the strength of the differential expressioin).
The very same (larger absolute) z-score can have different meanings, and this depends on the noise:</p>
<ul>
<li>with large noise: a very large effect</li>
<li>with some noise: a rather large effect</li>
<li>with only little noise: a rather small effect</li>
<li>with almost no noise: a tiny effect</li>
</ul>
</blockquote>
<blockquote>
<p>The problem for a biological interpretation is that &ldquo;noise&rdquo; is not only measurement noise. It is also connected to the &ldquo;tightness&rdquo; of the control of gene regulation. Genes that are not tightly controlled (so the expression may vary in a wider range) can be considerably induced or repressed and this would not be recognized by the z-scores. In contrast, genes that are tightly controlled may change the expression only very slighlty, without any biological impact, but they will get large absolute z-scores.</p>
</blockquote>

</main>

  <footer>
  <script src="//yihui.name/js/math-code.js"></script>
<script async src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML"></script>

<script async src="//yihui.name/js/center-img.js"></script>

  
  <hr/>
  © <a href="https://zijzhang.netlify.com">Zijian Zhang</a> 2018 &ndash; 2019 | <a href="https://github.com/zijzhang">Github</a>
  
  </footer>
  </body>
</html>

